commons/com.b2international.index.tests/src/com/b2international/index/revision/RevisionBranchMergeTest.java

```diff
@@ -99,6 +100,33 @@ public void fastForwardMergeBranchWithNewRevisionToParent() throws Exception {
 		assertNotNull(getRevision(MAIN, RevisionData.class, STORAGE_KEY1));
 	}
 	
+	@Test
+	public void forwardMergeBranchWithExclusions() throws Exception {
+		String child = createBranch(MAIN, "a");
+		// create a revisions on child branch
+		indexRevision(child, NEW_DATA);
+		indexRevision(child, NEW_DATA2);
+		// after commit child branch becomes FORWARD
+		assertState(child, MAIN, BranchState.FORWARD);
+		// do the merge with an exclusion
+		branching()
+			.prepareMerge(child, MAIN)
+			.setExclusions(ImmutableSet.of(STORAGE_KEY1))
```

Comment:
I would rename the `setExclusions()` method to just `exclude()` and give it a few overloaded versions:
```
exclude(String...ids)
exclude(Collection<String> ids)
```

Reply:
This suggestion is still not implemented, could you please add these as well?